tests/test_get_termURLs.py

```diff
@@ -0,0 +1,118 @@
+import pytest
+from requests.exceptions import RequestException
+from unittest.mock import patch
+from typing import List, Dict
+from app.fetch_termURLs.get_termURLs import (
+    fetch_termURL_mappings,
+    get_diagnosis_termURL,
+    get_assessment_termURL,
+    get_sex_termURL,
+    get_image_modal_termURL,
+)
+
+
+def test_fetch_termURL_mappings_success() -> None:
+    """
+    Test fetch_termURL_mappings function for successful API call.
+
+    Mocks a successful API response and checks if the returned data
+    matches the expected mock response data.
+    """
+    url = "http://example.com/api/mappings"
+    mock_response_data: List[Dict[str, str]] = [
+        {"TermURL": "termURL1", "Label": "label1"},
+        {"TermURL": "termURL2", "Label": "label2"},
+    ]
+
+    with patch("requests.get") as mock_get:
+        mock_get.return_value.status_code = 200
+        mock_get.return_value.json.return_value = mock_response_data
+
+        result = fetch_termURL_mappings(url)
+        assert result == mock_response_data
+
+
+def test_fetch_termURL_mappings_http_error() -> None:
+    """
+    Test fetch_termURL_mappings function for HTTP error.
+
+    Mocks an HTTP error during the API call and checks if the function
+    returns None.
+    """
+    url = "http://example.com/api/mappings"
+
+    with patch("requests.get") as mock_get:
+        mock_get.return_value.raise_for_status.side_effect = RequestException(
+            "HTTP Error"
+        )
+
+        result = fetch_termURL_mappings(url)
+        assert result is None
+
+
+@pytest.mark.parametrize(
+    "diagnosis, expected_termURL",
+    [
+        ("attention deficit hyperactivity disorder", "snomed:406506008"),
+        ("concussion injury of brain", "snomed:110030002"),
+        ("unknown disease", "None"),
+    ],
+)
+def test_get_diagnosis_termURL(diagnosis: str, expected_termURL: str) -> None:
```

Comment:
I think it's better to have tests not rely on an external API since if that API is down it will lead to tests failing without this app's code being responsible for the issue. So I suggest mocking the entire response of the api and see if the `get_diagnosis_termURL` works as expected.